frontend/packages/ux-editor/src/components/FormComponent.tsx

```diff
@@ -2,13 +2,13 @@ import React from 'react';
 import { useParams } from 'react-router-dom';
 import type { EditorDndEvents } from '../containers/helpers/dnd-types';
 import { EditFormComponent } from './EditFormComponent';
-import type { FormComponentType } from '../types/global';
 import { DEFAULT_LANGUAGE } from 'app-shared/constants';
 import { useFormLayoutsSelector } from '../hooks/useFormLayoutsSelector';
 import { selectedLayoutSelector } from '../selectors/formLayoutSelectors';
 import { ComponentType } from './index';
 import { useTextResourcesQuery } from '../../../../app-development/hooks/queries/useTextResourcesQuery';
 import { DroppableDraggableComponent } from '../containers/DroppableDraggableComponent';
+import type { FormComponent as IFormComponent } from '../types/FormComponent';
```

Comment:
I noticed that you removed both the prefix (IFormComponent) and the suffix (FormComponentType). I think it's a good idea since it makes it more readable but it seems that it can also lead to name conflicts from time to time, like here, so I'm not 100% sure what the best solution is. But then, I was wondering, do we have a naming convention for types / interfaces? 🤔

Reply:
Fair point, I was also uncertain about this. `FormComponentType` is surely not a good name since the interface describes a component, not a type. The "I" prefix is used somewhere in Studio, but not everywhere, so I found it better to remove it, since it's also not recommended (just google "i prefix typescript"). Hopefully it will not lead to many naming collisions like this one, but if it does, we should definitely agree upon some convention!